themes/src/main/resources/theme/base/login/resources/js/webauthnAuthenticate.js

```diff
@@ -0,0 +1,72 @@
+import { base64url } from "../../../common/keycloak/node_modules/rfc4648/lib/rfc4648.js";
+
+window.authenticateByWebAuthn = (isUserIdentified, challenge, userVerification, rpId, createTimeout, errmsg) => {
+    if (!isUserIdentified) {
+        doAuthenticate([], challenge, userVerification, rpId, createTimeout, errmsg);
+        return;
+    }
+    checkAllowCredentials(challenge, userVerification, rpId, createTimeout, errmsg);
+}
+
+function checkAllowCredentials(challenge, userVerification, rpId, createTimeout, errmsg) {
+    let allowCredentials = [];
+    let authn_use = document.forms['authn_select'].authn_use_chk;
```

Comment:
This can also be a constant, and let's keep the casing of variables consistent.
```suggestion
    const authnUse = document.forms['authn_select'].authn_use_chk;
```

Reply:
I fixed the codes as you suggested.